invenio_rdm_records/requests/access/requests.py

```diff
@@ -200,4 +234,5 @@ def _update_link_config(self, **context_vars):
         "full_name": fields.String(required=True),
         "token": fields.String(required=True),
         "message": fields.String(required=False),
+        "secret_link_expiration": fields.String(required=True, validate=_validate_date),
```

Comment:
see comment above. We could avoid the `required=True`, and have a `pre_load` hook to set it to a specific value if needed.